src/transform/class/PotentialClass.js

```diff
@@ -93,6 +100,38 @@ class PotentialClass {
   }
 
   createConstructor() {
-    return this.constructor.isEmpty() ? undefined : this.constructor.toMethodDefinition();
+    if (this.constructor.isEmpty()) {
+      return undefined;
+    }
+    else {
+      this.modifySuperCalls();
+      return this.constructor.toMethodDefinition();
+    }
+  }
+
+  modifySuperCalls() {
+    this.constructor.methodNode.body.body.forEach(body => {
```

Comment:
This code also expects that the call to superclass is directly inside the body of the function. But what if it's somewhere deeper like:

```js
function MyClass() {
  if (someCondition) {
    SuperClass.call(this, name);
  }
}
```